app/src/main/java/nu/yona/app/api/model/YonaDevice.java

```diff
@@ -16,6 +16,10 @@
 	@Expose
 	private String sslRootCertCN;
 
+	@SerializedName("vpnProfile")
+	@Expose
+	private VpnProfile vpnProfile;
```

Comment:
You need to increment the version number of the user data, as older versions don't have the VPN profile.